docs/lang/const.md

```diff
@@ -1,25 +1,25 @@
-C++ 定义了一套完善的只读量定义方法，被常量修饰符 `const` 修饰的对象或类型都是只读量，只读量的内存存储与一般变量没有任何区别，但是编译器会在编译期进行冲突检查，避免对只读量的修改。因此合理使用 `const` 修饰符可以增加代码健壮性。
+C++ defines a complete set of methods for defining read-only variables. Objects or types modified by the constant modifier `const` are all read-only. The memory storage of read-only variables is no different from general variables, but the compiler will compile periodically check for conflicts to avoid modification of read-only variables. Therefore, reasonable use of the `const` modifier can increase code robustness.
 
-## 常类型
+## Regular type
 
-在类型的名字前增加 const 修饰会将该类型的变量标记为不可变的。具体使用情况有常量和常引用（指针）两种。
+Adding a const modification before the name of a type will mark the variable of that type as immutable. There are two types of specific usage: constant and constant reference (pointer).
 
-### 常量
+### Constant
 
-这里的常量即常变量，指的是 const 类型的变量（而不是标题里泛指的只读量）。常类型量在声明之后便不可重新赋值，也不可访问其可变成员，只能访问其常成员。常成员的定义见后文。
+The constant here is a constant variable, which refers to a variable of const type (not a read-only variable we generally refer to in the title). The constant type cannot be reassigned after declaration, nor can it access its variable members, only its constant members. See below for the definition of constant members.
```

Comment:
```suggestion
The constant here is a constant variable, which refers to a variable of const type (instead of read-only variable we generally refer to in the title). The constant-typed variables cannot be reassigned after declaration, nor can it access its mutable members, only its constant members. See the text below for the definition of constant members.
```